mithril-aggregator/src/http_server/routes/signatures_routes.rs

```diff
@@ -146,12 +147,45 @@ mod tests {
         .unwrap();
     }
 
+    #[tokio::test]
+    async fn test_register_signatures_post_ok_202() {
```

Comment:
Yes, it's implicit since in the code and test we use enum variant with the explicit name of the http code.

Until now this is how we have named those tests has it has the benefit of clarity as long as you have some knowledge of the http code numbers. A requirement that can be though as reasonable since this whole part is an http api.